### PR TITLE
Update importdialog.js

### PR DESCRIPTION
**Pull request by TheMitch2**
Change to update for MPE in IITC

### CODE REVIEW

**Comment by TheMitch2** [comment]
Unable to link to issue #260

File: src/code/dialogs/importDialog.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ const ImportDialog = WDialog.extend({
   },
 
   drawToolsFormat() {
-    const dtitems = localStorage["plugin-draw-tools-layer"];
+    const dtitems = localStorage["window.plugin.drawTools.KEY_STORAGE"];
```
Comment (cloudkucooland) [inline src/code/dialogs/importDialog.js:73]: remove the quotes, it is not a static string, it is a variable.

Comment (TheMitch2) [reply src/code/dialogs/importDialog.js]: Updated

**Review by cloudkucooland — COMMENTED** [review]
what does this do if drawTools is not installed? Make sure to check with the console open

**Comment by TheMitch2** [comment]
I created a simple draw and the removed the DT plugin. Import from DT worked fine with my build.

**Comment by TheMitch2** [comment]
Nothing yellow or red in console

**Comment by le-jeu** [comment]
> I created a simple draw and the removed the DT plugin. Import from DT worked fine with my build.

it shouldn't  since `window.plugin.drawTools` is undefined when DT is not enable (at plugin level, not as a layer)

**Comment by TheMitch2** [comment]
Okay, i need to add that then. Later i can figure out why the build was not based on the code I expected.

**Review by le-jeu — APPROVED** [review]

**Pull request merged**